public/js/user-actions.js

```diff
@@ -13,13 +16,56 @@ function UserActions(localDispatcher) {
 }
 
 
-UserActions.prototype.setCurrentUser = function(user) {
-  console.log('UserActions: setting current user', user);
-  this.dispatcher.dispatch({
-    actionType: 'set-user',
-    user: user,
-  });
-};
+UserActions.prototype = assign({}, {
```

Comment:
I don't understand the point of using assign here. Isn't this literally the exact same as `UserActions.prototype = {`?